inginious/frontend/i18n/es/LC_MESSAGES/messages.po

```diff
@@ -68,41 +68,87 @@ msgstr ""
 msgid "<p>Custom feedback</p><pre>{custom_feedback}</pre><br>"
 msgstr "<p>Retroalimentación personalizada</p><pre>{custom_feedback}</pre><br>"
 
-#: inginious/frontend/parsable_text.py:202
+#: inginious/frontend/parsable_text.py:204
 msgid "<b>Parsing failed</b>: <pre>{}</pre>"
 msgstr "<b>El parsing ha fallado</b>: <pre>{}</pre>"
 
-#: inginious/frontend/parsable_text.py:230
+#: inginious/frontend/parsable_text.py:246
 msgid ""
 "**Compilation error**:\n"
 "\n"
 msgstr ""
 "**Error en compilación**:\n"
 "\n"
 
-#: inginious/frontend/parsable_text.py:232
+#: inginious/frontend/parsable_text.py:248
 msgid ""
 "<br><strong>{}:</strong> There was an error while running your notebook: "
 "<br><pre>{}</pre><br>"
 msgstr ""
 "<br><strong>{}:</strong> Hubo un error mientras se ejecutaba el notebook:"
 " <br><pre>{}</pre><br>"
 
-#: inginious/frontend/parsable_text.py:357
+#: inginious/frontend/parsable_text.py:366
+msgid ""
+"<ul class=\"list_disc\" style=\"font-size:12px;\"><li>\n"
+"            <strong style=\"font-size:15px\"> Test{test_id}: </strong><i>"
+" - YOUR GRADE = {test_grade} / 100 </i>"
+msgstr ""
+"<ul class=\"list_disc\" style=\"font-size:12px;\"><li>\n"
+"            <strong style=\"font-size:15px\"> Test{test_id}: </strong><i>"
+" - Calificación = {test_grade} / 100 </i>"
+
+#: inginious/frontend/parsable_text.py:371
+msgid ""
+"<a class=\"btn btn-default btn-link btn-xs\" role=\"button\"\n"
+"            data-toggle=\"collapse\" href=\"#{panel_id}\" aria-"
+"expanded=\"false\" aria-controls=\"{panel_id}\">\n"
+"            View Details </a><div class=\"collapse\" id=\"{panel_id}\"> "
+"<pre>{test_message}</pre> <br>\n"
+"            <ul class=\"list_disc\" style=\"font-size:13px;\">"
+msgstr ""
+"<a class=\"btn btn-default btn-link btn-xs\" role=\"button\"\n"
+"            data-toggle=\"collapse\" href=\"#{panel_id}\" aria-"
+"expanded=\"false\" aria-controls=\"{panel_id}\">\n"
+"            Ver detalles </a><div class=\"collapse\" id=\"{panel_id}\"> "
+"<pre>{test_message}</pre> <br>\n"
+"            <ul class=\"list_disc\" style=\"font-size:13px;\">"
+
+#: inginious/frontend/parsable_text.py:378
+msgid "<li><strong style=\"font-size:14px\"> FUNCTIONS </strong> <br>"
+msgstr "<li><strong style=\"font-size:14px\"> FUNCIONES </strong> <br>"
+
+#: inginious/frontend/parsable_text.py:381
+msgid "<li><strong style=\"font-size:14px\"> VARIABLES </strong> <br>"
+msgstr "<li><strong style=\"font-size:14px\"> VARIABLES </strong> <br>"
+
+#: inginious/frontend/parsable_text.py:391
+#: inginious/frontend/parsable_text.py:399
+#, fuzzy
+msgid ""
+"<strong>: </strong><pre class=\"language-python\"><code\n"
+"                class=\"language-python\" data-"
+"language=\"python\"></code></pre><script>highlight_code();</script>"
+msgstr ""
+"<strong>Código ejecutado:</strong><pre class=\"language-python\"><code "
+"class=\"language-python\" data-"
+"language=\"python\">{case_code}</code></pre><script>highlight_code();</script>"
+
```

Comment:
Careful here, normally translations with the fuzzy tag mean that pybabel is not sure about it, the best practice is check fuzzy tags and fix them, here for example in the translation, there is an extra parameter {case_code} on line 135 that is not on the msgid, remove it and check if it works without problem